### PR TITLE
Adding a number component to the wallbox integration.

### PR DESCRIPTION
**Pull request by hesselonline**
Related to PR .... in homeassistant/core -> adding a number component to the wallbox integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/52786
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR, probably caused by using the `current` branch as the base branch. Please git rebase this PR onto the latest `next` branch to clean it up.

Thanks! 👍 

../Frenck

File: source/_integrations/wallbox.markdown [hunk 1]
```diff
@@ -32,3 +33,10 @@ The integration adds the following sensors:
 - Max Available Power
 - State of Charge
 - Status Description
+- Max Charging Current
+
+## Number
+
+The integration adds the following number (input) component:
```
Comment (frenck) [inline source/_integrations/wallbox.markdown]: `input` is for manual helpers, component is a term we do not use for user-facing documentation.

Suggestion:

```suggestion
The integration adds the following number entity:
```

Comment (hesselonline) [reply source/_integrations/wallbox.markdown]: I changed it into entity.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]
Thanks, @hesselonline 👍

**Pull request merged**